.gitattributes

```diff
@@ -0,0 +1,20 @@
+# From: https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.rs text eol=lf
+*.toml text eol=lf
+*.frag text eol=lf
+*.vert text eol=lf
+*.wgsl text eol=lf
+*.md text eof=lf
+
+# Declare files that will always have CRLF line endings on checkout.
+*.sln text eol=crlf
```

Comment:
I don't think we need this. I doubt we'll ever have a .sln in this repo

Reply:
I agree, fixed in e32cf6a14d21b404ff09af0a405af4c986d86795.